test/e2e/testing-manifests/storage-csi/mock/csi-mock-driver.yaml

```diff
@@ -53,10 +53,17 @@ spec:
           - mountPath: /registration
             name: registration-dir
         - name: mock
-          image: k8s.gcr.io/sig-storage/mock-driver:v4.1.0
+          # FIXME: Move to release image once the hostpath-plugin release
+          # having below chagnes;
+          #  - https://github.com/kubernetes-csi/csi-driver-host-path/pull/269
+          #  - https://github.com/kubernetes-csi/csi-driver-host-path/pull/260          #
+          image: k8s.gcr.io/sig-storage/hostpathplugin:canary
           args:
-            - "--name=mock.storage.k8s.io"
-            - "-v=3" # enabled the gRPC call logging
+            - "--drivername=mock.storage.k8s.io"
+            - "--nodeid=$(KUBE_NODE_NAME)"
+            - "--endpoint=/csi/csi.sock"
+            - "--statedir=/csi-data-dir"
```

Comment:
My proposal was to use `/tmp/csi-hostpath-data`, not `/csi-data-dir`. It makes the intent clearer (don't persist) and might also be a bit more efficient (tmpfs vs overlayfs with actual disk IO).

Reply:
Changed statedir to /tmp/csi-hostpath-data, as suggested.